bosh-templates/cloud_controller_api.yml.erb

```diff
@@ -187,6 +187,12 @@ uaa:
 hm9000:
   url: <%= p("hm9000.url", "https://hm9000.#{domain}") %>
 
+<% if p("router.enable_routing_api") %>
+routing_api:
+  url: <%= "https://api.#{domain}/routing" %>
```

Comment:
How is this supposed to work?  Does this imply the routing-api must be colocated with all the CCs?  Won't nginx send all requests to the CC process, how will the routing-api actually get any requests with this URL?